[0.050s] Invoking command in '/home/minha/myagv_ros2/myagv_ros2/build/myagv_cartographer': CONDA_PROMPT_MODIFIER=(new_adaface) /home/minha/anaconda3/envs/adaface/bin/cmake /home/minha/myagv_ros2/myagv_ros2/src/myagv_cartographer -DCMAKE_INSTALL_PREFIX=/home/minha/myagv_ros2/myagv_ros2/install/myagv_cartographer
[0.117s] -- Found ament_cmake: 1.3.8 (/opt/ros/humble/share/ament_cmake/cmake)
[0.268s] -- Found ament_lint_auto: 0.12.10 (/opt/ros/humble/share/ament_lint_auto/cmake)
[0.384s] Error parsing '/home/minha/myagv_ros2/myagv_ros2/src/myagv_cartographer/package.xml':
[0.384s] Traceback (most recent call last):
[0.384s]   File "/home/minha/anaconda3/envs/adaface/lib/python3.9/site-packages/catkin_pkg/package.py", line 610, in parse_package_string
[0.384s]     root = dom.parseString(data)
[0.384s]   File "/home/minha/anaconda3/envs/adaface/lib/python3.9/xml/dom/minidom.py", line 1998, in parseString
[0.384s]     return expatbuilder.parseString(string)
[0.384s]   File "/home/minha/anaconda3/envs/adaface/lib/python3.9/xml/dom/expatbuilder.py", line 925, in parseString
[0.384s]     return builder.parseString(string)
[0.384s]   File "/home/minha/anaconda3/envs/adaface/lib/python3.9/xml/dom/expatbuilder.py", line 223, in parseString
[0.384s]     parser.Parse(string, True)
[0.385s] xml.parsers.expat.ExpatError: mismatched tag: line 12, column 33
[0.385s] 
[0.385s] During handling of the above exception, another exception occurred:
[0.385s] 
[0.385s] Traceback (most recent call last):
[0.385s]   File "/opt/ros/humble/share/ament_cmake_core/cmake/core/package_xml_2_cmake.py", line 150, in <module>
[0.385s]     main()
[0.385s]   File "/opt/ros/humble/share/ament_cmake_core/cmake/core/package_xml_2_cmake.py", line 53, in main
[0.385s]     raise e
[0.385s]   File "/opt/ros/humble/share/ament_cmake_core/cmake/core/package_xml_2_cmake.py", line 49, in main
[0.385s]     package = parse_package_string(
[0.385s]   File "/home/minha/anaconda3/envs/adaface/lib/python3.9/site-packages/catkin_pkg/package.py", line 612, in parse_package_string
[0.385s]     raise InvalidPackage('The manifest contains invalid XML:\n%s' % ex, filename)
[0.385s] catkin_pkg.package.InvalidPackage: Error(s) in package '/home/minha/myagv_ros2/myagv_ros2/src/myagv_cartographer/package.xml':
[0.386s] The manifest contains invalid XML:
[0.386s] mismatched tag: line 12, column 33
[0.397s] [31mCMake Error at /opt/ros/humble/share/ament_cmake_core/cmake/core/ament_package_xml.cmake:95 (message):
[0.397s]   execute_process(/home/minha/anaconda3/envs/adaface/bin/python3
[0.398s]   /opt/ros/humble/share/ament_cmake_core/cmake/core/package_xml_2_cmake.py
[0.398s]   /home/minha/myagv_ros2/myagv_ros2/src/myagv_cartographer/package.xml
[0.398s]   /home/minha/myagv_ros2/myagv_ros2/build/myagv_cartographer/ament_cmake_core/package.cmake)
[0.398s]   returned error code 1
[0.398s] Call Stack (most recent call first):
[0.398s]   /opt/ros/humble/share/ament_cmake_core/cmake/core/ament_package_xml.cmake:49 (_ament_package_xml)
[0.398s]   /opt/ros/humble/share/ament_lint_auto/cmake/ament_lint_auto_find_test_dependencies.cmake:31 (ament_package_xml)
[0.398s]   CMakeLists.txt:26 (ament_lint_auto_find_test_dependencies)
[0.398s] 
[0.398s] -- Configuring incomplete, errors occurred!
[0.398s] [0m
[0.425s] Invoked command in '/home/minha/myagv_ros2/myagv_ros2/build/myagv_cartographer' returned '1': CONDA_PROMPT_MODIFIER=(new_adaface) /home/minha/anaconda3/envs/adaface/bin/cmake /home/minha/myagv_ros2/myagv_ros2/src/myagv_cartographer -DCMAKE_INSTALL_PREFIX=/home/minha/myagv_ros2/myagv_ros2/install/myagv_cartographer
